README.md

```diff
@@ -1,16 +1,1261 @@
-# UCAN Invocation
+# UCAN Invocation Specification v0.1.0
```

Comment:
```suggestion
# UCAN Invocation Specification v0.1.0-draft
```

Can we call it a draft for now ?

Reply:
Is that substantially different from a pre-1.0 release? From the semver FAQ:

> Major version zero is all about rapid development. If you’re changing the API every day you should either still be in version 0.y.z or on a separate development branch working on the next major version.

Reply:
For those following along: we covered this live on a call on Friday, and `0.X.Y` probably communicates this sufficiently